src/app/shell/sidebar/Sidebar.css

```diff
@@ -9,8 +9,12 @@
 }
 
 .sidebar {
-    position: relative;
+    position: fixed;
```

Comment:
As I said earlier, I think the position of the sidebar should be defined in the Shell.css file. It's the role of the Shell to position the content and the sidebar correctly, including their z-index.

You may leave the `position: relative` here because there's elements inside the sidebar that are positioned relative to it, but the `position: fixed` defined in the shell will win.